mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1937,6 +1937,28 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+			"fleet": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				MaxItems:    1,
+				Description: `Fleet information of the cluster.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"project": {
```

Comment:
I see `This field cannot be changed after the cluster has been registered.` in the docs for this field - that seems to imply it's _not_ mutable, but the PR description explicitly says that it _is_ mutable - why does there seem to be a difference? (I suspect I might be misunderstanding something.)